### PR TITLE
added vue-chess-storybook example

### PR DESCRIPTION
**Pull request by gustaYo**
Refactoring components to Vue2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -339,6 +339,8 @@ vue-router 2.0, vue-infinite-scroll 2.0, vue-progressbar 2.0 by [TIGERB](https:/
   - [route-planner-vue](https://kasheftin.github.io/route-planner-vue/) by [@Kasheftin](https://github.com/Kasheftin): The tool for planning routes with multiple sortable layers, draggable directions, markers and shapes on google map.
   - [MyDiary-Vue](https://github.com/ssshooter/MyDiary-Vue/blob/master/README.En.md) A diary application build with Vue 2.X which is also have contact and todolist function
   - [VueJS Example Projects](https://github.com/vue-project) on Github by [Ali GÖREN](https://aligoren.com)
+  - [vue-chess-storybook](https://github.com/gustaYo/vue-chess-storybook) -
```
Comment (nickmessing) [inline README.md]: Kepp this in a single line without `\n` please.

File: README.md [hunk 2]
```diff
@@ -339,6 +339,7 @@ vue-router 2.0, vue-infinite-scroll 2.0, vue-progressbar 2.0 by [TIGERB](https:/
   - [route-planner-vue](https://kasheftin.github.io/route-planner-vue/) by [@Kasheftin](https://github.com/Kasheftin): The tool for planning routes with multiple sortable layers, draggable directions, markers and shapes on google map.
   - [MyDiary-Vue](https://github.com/ssshooter/MyDiary-Vue/blob/master/README.En.md) A diary application build with Vue 2.X which is also have contact and todolist function
   - [VueJS Example Projects](https://github.com/vue-project) on Github by [Ali GÖREN](https://aligoren.com)
+  - [vue-chess-storybook](https://github.com/gustaYo/vue-chess-storybook) -Refactoring components at Vue2 by [gustaYo](https://github.com/gustaYo)
```
Comment (nickmessing) [inline README.md]: > - [vue-chess-storybook](https://github.com/gustaYo/vue-chess-storybook) -Refactoring components at Vue2 by [gustaYo](https://github.com/gustaYo)

This doesn't look clear enough IMO, maybe something like
> - [Chess Storybook Example](https://github.com/gustaYo/vue-chess-storybook) with Vue 2.0 by [gustaYo](https://github.com/gustaYo)

?

**Comment by nickmessing** [comment]
@kazupon, it's in `Examples` category.

**Comment by kazupon** [comment]
@nickmessing ah, sorry 🙇

**Pull request merged**